src/Collections-Tests/ByteSymbolTest.class.st

```diff
@@ -9,76 +9,63 @@ Class {
 
 { #category : #'tests - creation' }
 ByteSymbolTest >> testAs [
-	"self debug: #testAs"
-
-| tStr tAs1 tAs2 |
+	
+	| tStr tAs1 tAs2 |
 
-tStr := DateAndTime now asString .
-tAs1 :=  tStr as: ByteSymbol . 
-self assert: 	(Symbol allSymbols select: [:e | e asString  = tStr ]  ) size = 1 .
-self assert: 	(ByteSymbol allInstances select: [:e | e asString = tStr] ) size = 1 .  
-self assert: 	(ByteSymbol allInstances select: [:e | e asString = tStr] ) =
+	tStr := DateAndTime now asString .
+	tAs1 :=  tStr as: ByteSymbol . 
+	self assert: (Symbol allSymbols select: [:e | e asString  = tStr ]  ) size = 1 .
+	self assert: (ByteSymbol allInstances select: [:e | e asString = tStr] ) size = 1 .  
+	self assert: (ByteSymbol allInstances select: [:e | e asString = tStr] ) =
 		(Symbol allSymbols select: [:e | e asString  = tStr ]  ) .
 		
-tAs2 :=  tStr as: ByteSymbol . 
-self assert: 	(Symbol allSymbols select: [:e | e asString  = tStr ]  ) size = 1 .
-self assert: 	(ByteSymbol allInstances select: [:e | e asString = tStr] ) size = 1 .  
+	tAs2 := tStr as: ByteSymbol . 
+	self assert: (Symbol allSymbols select: [:e | e asString  = tStr ]  ) size = 1 .
+	self assert: (ByteSymbol allInstances select: [:e | e asString = tStr] ) size = 1 .  
 
 
 ]
 
 { #category : #'tests - non-creation' }
 ByteSymbolTest >> testNew [
-	"self debug: #testNew"
-
 	
 	self should: [ByteSymbol new: 5 ] raise: self defaultTestError. 
 ]
 
 { #category : #'tests - creation' }
 ByteSymbolTest >> testNewFrom [
-	"self debug: #testNewFrom"
-
-| dt newFrom1 newFrom2 |
-
-dt := DateAndTime now asString .
-newFrom1 := ByteSymbol newFrom: dt .
-
-self assert:
-	(Symbol allSymbols select: [:e | e asString  = dt  ]  ) size = 1 .
-self assert:
-	(ByteSymbol allInstances select: [:e | e = dt  ] ) size = 1 .
-self assert:
-	(Symbol allSymbols select: [:e | e asString  = dt  ]  ) =
-			(ByteSymbol allInstances select: [:e | e = dt ] ) .
-			
-newFrom2 := ByteSymbol newFrom: dt . 
-self assert:
-	(Symbol allSymbols select: [:e | e asString  = dt  ]  ) size = 1 .
-self assert:
-	(ByteSymbol allInstances select: [:e | e = dt ] ) size = 1 .
-	
+	| dt newFrom1 newFrom2 |
+	dt := DateAndTime now asString.
+	newFrom1 := ByteSymbol newFrom: dt.
+	self assert: (Symbol allSymbols select: [ :e | e asString = dt ]) size = 1.
+	self assert: (ByteSymbol allInstances select: [ :e | e = dt ]) size = 1.
```

Comment:
equals: would be nice but let's do it with some next PR

Reply:
I do them separately in an own pass